internal/api/impl/v1/datasource/service.go

```diff
@@ -124,7 +124,12 @@ func (s *service) Get(parameters shared.Parameters) (interface{}, error) {
 }
 
 func (s *service) List(q etcd.Query, _ shared.Parameters) (interface{}, error) {
-	return s.dao.List(q)
```

Comment:
Does this branch need to be rebased (seeing a lot of .go file changes)?

Reply:
Yup, I just screwed up a merge. Working on it.

Reply:
OK, should be good to go now! Git is hard. 😂